form.html

```diff
@@ -0,0 +1,207 @@
+                  <input class="form__input-date" id="checkin" name="checkin" type="date">
+                </div>
+                <div class="form__stay">
+                  <label class="form__label-number-tablet" for="stay">DAYS OF STAY<p class="form__important">*</p></label>
+                  <label class="form__label-number" for="stay">LENGTH OF STAY<p class="form__important">*</p></label>
+                  <div class="form__input-inc"></div>
+                  <input class="form__input-number" id="stay" min="1" step="1" max="31" placeholder="day" type="number">
+                  <div class="form__input-dec"></div>
+                </div>
+                <div class="form__checkout">
+                  <label class="form__label-date" for="checkout">CHECKOUT:<p class="form__important">*</p></label>
+                  <input class="form__input-date" id="checkout" name="checkout" type="date">
+                </div>
+            </div>
+          </section>
+
+          <section class="form__section">
+            <div class="form__travellers">
+              <h2 class="form__section-header">NUMBER OF TRAVELLERS</h2>
+              <div class="form__ammount">
+                <label class="form__label-number" for="stay">NUMBER OF TRAVELLERS:<p class="form__important">*</p></label>
+                <div class="form__input-inc"></div>
+                <input class="form__input-number" id="stay" min="1" step="1" max="10" plceholder="person" type="number">
+                <div class="form__input-dec"></div>
+              </div>
+              <div class="form__person">
+                <div class="form__count"></div>
+                <div class="form__person-surname">
+                  <label class="form__label-text" for="person-surname">Surname:</label>
+                  <input class="form__input-text" id="person-surname" name="person-surname" type="text">
+                </div>
+                <div class="form__person-name">
+                  <label class="form__label-text" for="person-name">Name:</label>
+                  <input class="form__input-text" id="person-name" name="name" type="text">
+                </div>
+                <div class="form__person-middle-name">
+                  <label class="form__label-text" for="person-second-name">Middle name:</label>
+                  <input class="form__input-text" id="person-second-name" name="person-second-name" type="text">
+                </div>
+              </div>
+            </div>
+          </section>
+
+          <section class="form__section">
+            <div class="form__photos">
+              <h2 class="form__header">UPLOAD PHOTOS</h2>
+              <div>
+                <label class="form__label-file" for="file">CHOOSE PHOTOS</label>
+                <input class="form__input-file" id="file" name="file" accept=".jpg, .jpeg, .png" type="file" multiple>
+              </div>
+              <div class="form__display-photos">
```

Comment:
You still need to add Items inside. The idea is that:
1) You create layout for everything and add images here manually
2) We use added item as a template for javascript late.